cpu/nrf52/Makefile.include

```diff
@@ -35,6 +35,30 @@ ifneq (,$(filter nrf52840xxaa,$(CPU_MODEL)))
   RAM_LEN ?= 0x40000
 endif
 
+# Set the target for the JLink programmer for flashing.
+# Boards that do not use a JLink ignore the flag.
+ifneq (,$(filter nrf52805xxaa,$(CPU_MODEL)))
+  JLINK_DEVICE = nrf52805_xxaa
+endif
+ifneq (,$(filter nrf52810xxaa,$(CPU_MODEL)))
+  JLINK_DEVICE = nrf52810_xxaa
+endif
+ifneq (,$(filter nrf52811xxaa,$(CPU_MODEL)))
+  JLINK_DEVICE = nrf52811_xxaa
+endif
+ifneq (,$(filter nrf52820xxaa,$(CPU_MODEL)))
+  JLINK_DEVICE = nrf52820_xxaa
+endif
+ifneq (,$(filter nrf52832xxaa,$(CPU_MODEL)))
+  JLINK_DEVICE = nrf52832_xxaa
+endif
+ifneq (,$(filter nrf52833xxaa,$(CPU_MODEL)))
+  JLINK_DEVICE = nrf52833_xxaa
+endif
+ifneq (,$(filter nrf52840xxaa,$(CPU_MODEL)))
+  JLINK_DEVICE = nrf52840_xxaa
+endif
```

Comment:
how about reducing this to:
```mk
JLINK_DEVICE = $(subst xx,_xx,$(CPU_MODEL))
```
?

Reply:
Sure. I'll change it tomorrow.

The only risk is that at some point the naming scheme of RIOT and JLINK would deviate for a new nRF52 device. But with either solution that would require a change, so the risk is probably negligible.